### ISSUE TITLE
Some form of automation or installer (probably require administrator rights)

### ISSUE THREAD

**Issue by bonzibudd** [bonzibudd/Fluent-Metro#61]
The skin should have some sort of automated installer, to accomplish these tasks:
 - Prompt user with installation options (Which skins to install, to apply XML for tiles or not)
 - Ask for admin rights to copy files
 - Show success or failure
 - If success, prompt final message
 - If failure, provide alternative method of installing

All developments will be added to this thread.

**Comment by bonzibudd** [comment]
After some research, I determined that it will be possible to do most of the things I need with Batch, however I’m not sure about how it works with the admin prompt. It may work automatically, but I haven’t gotten to the point of testing it, other than a few individual copy commands.

**Comment by jonathan-rogers-dev** [comment]
Has any progress been made on this?

**Comment by bonzibudd** [comment]
Yes. I found the syntax for the syntax for applying the [XML file](http://www.classicshell.net/forum/viewtopic.php?p=28499), and that appears work alongside the copy commands. I have been testing everything with batch, and I haven’t quite figured out the structure of it yet, but I did have some success for checking if OSM is installed/which Windows version is being used. Nothing has been finalized though, and I might end up going with PowerShell if it can do what I need more effectively. I don’t know when it will be done, but I might prioritize it if it seems to be worth going forward with.
